.github/workflows/lint.yml

```diff
@@ -1,13 +1,14 @@
 name: Lint
 
 on:
-  # Trigger the workflow on push or pull request,
-  # but only for the main branch
   push:
     branches: [main]
   pull_request:
-    branches:
-      - "*"
```

Comment:
Why is this deleted? Is this the default if not specified?

Reply:
Not exactly. Actually we need to remove it or change it to `**`. Per github action doc (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), `*` match zero or more characters except "/", but `**` match everything. And you're right, if branches is not specified, there will be no filter on it.